lib/unhangout-routes.js

```diff
@@ -788,6 +795,136 @@ module.exports = {
         // Starting and stopping events
         //
 
+        app.get("/followup/event/:id/participant_:participantCount", function(req, res) {
+
+            var event = db.events.get(req.params.id);
+
+            var eventTitle = event.get("title");
+            var history = event.get("history");
+            var participantCount = req.params.participantCount; 
+
+            var allUserIds = [];
+            allUserIds.push(null);
```

Comment:
This variable should be named better.  It does not contain userIds; it contains ``{user: user, users: users}`` object.  Maybe call it "userData" or ``recipientData``.

Also: Why is this prepended with ``null``?